internal/components/processors_test.go

```diff
@@ -199,6 +200,31 @@ func verifyProcessorLifecycle(t *testing.T, factory processor.Factory, getConfig
 	}
 }
 
+// verifyProcessorShutdown is used to test if a processor type can be shutdown without being started first.
+// We disregard errors being returned by shutdown, we're just making sure the processors don't panic.
+func verifyProcessorShutdown(_ *testing.T, factory processor.Factory, getConfigFn getProcessorConfigFn) {
```

Comment:
Any reason for ignoring the `*testing.T` here?

Reply:
We don't do anything with it.